packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -1330,6 +1360,7 @@ Original error: ${e.message}
       }
       return build.extend(build, {
         pgIntrospectionResultsByKind: introspectionResultsByKind,
+        pgMakeFakeEnumIdentifier: fakeEnumIdentifier,
```

Comment:
I'm okay with this name, but I don't usually use `Make` for scalars. I'd probably have called it `getPgFakeEnumIdentifier` or similar. (No action necessary.)